src/Riverline/MultiPartParser/Part.php

```diff
@@ -46,7 +46,7 @@ public function __construct($content)
     protected function parseContentString($content)
     {
         // Split headers and body
-        $splits = preg_split('/(\r?\n){2}/', $content, 2);
```

Comment:
I was thinking a bit about it, can you just add `preg_replace('/\r?\n/', "\r\n", $string)` before throwing data into parser? This would allow to avoid overhead inside of the library, but will indeed work in your case as well.
Regular expressions will be especially costly if you, for instance, processing 2 GiB stream with uploaded files.